common/src/dodeca.rs

```diff
@@ -334,6 +346,39 @@ mod data {
         })
     }
 
+    // Which transformations have to be done after a reflection to switch reference frames from one vertex
+    // to one of its adjacent vertices (ordered similarly to ADJACENT_VERTICES)
```

Comment:
Is "adjacent vertices" the right term here? What about the chunk associated with the same vertex in an adjacent dodecahedron?

Reply:
I believe it should be. They're vertices in a single dodecahedron that share an edge. Since the term used is "adjacent vertices" and not "vertices of adjacent chunks", I don't think it's ambiguous.